Test/New-WhiskeyContext.Tests.ps1

```diff
@@ -621,30 +627,35 @@ InModuleScope -ModuleName 'Whiskey' -ScriptBlock {
         ThenSemVer1Is '1.2.3'
     }
 
-    function GivenPackageJson
+    function GivenNodeVersionPath
+    {
+        param(
+            [string]
+            $AtVersion,
+            [string]
+            $withPath
+        )
+        Mock -CommandName 'New-WhiskeySemanticVersion' -ModuleName 'Whiskey' -ParameterFilter {$path} -MockWith { return [SemVersion.SemanticVersion]$AtVersion }.GetNewClosure()

```

Comment:
Why mock this? Why not create an actual `package.json` file on the file system?

Reply:
Done, I originally mocked it because it was being mocked by the other tests. but this made it a better test! thanks